From vapier@gentoo.org  Tue Jun 23 14:59:34 2009
From: Mike Frysinger <vapier@gentoo.org>
Date: Thu, 11 Jun 2009 21:59:00 -0400
Subject: USB: isp1760: use __devexit_p() for remove function
To: linux-usb@vger.kernel.org
Message-ID: <1244771940-15276-1-git-send-email-vapier@gentoo.org>


The isp1760_plat_remove function is declared with __devexit, so the
.remove assignment needs to be wrapped with __devexit_p().

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/isp1760-if.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/isp1760-if.c
+++ b/drivers/usb/host/isp1760-if.c
@@ -361,7 +361,7 @@ static int __devexit isp1760_plat_remove
 
 static struct platform_driver isp1760_plat_driver = {
 	.probe	= isp1760_plat_probe,
-	.remove	= isp1760_plat_remove,
+	.remove	= __devexit_p(isp1760_plat_remove),
 	.driver	= {
 		.name	= "isp1760",
 	},
